install/updates/froxlor/0.9/update_0.9.inc.php

```diff
@@ -3893,6 +3893,12 @@
 	updateToVersion('0.9.39.2');
 }
 
+if (isFroxlorVersion('0.9.39.2-dev1')) {
```

Comment:
wrong position, always add changes to the END of the update file. Also, as said earlier, just adjust the db_version and use isDatabaseVersion()

Reply:
removed